packages/token/src/tests/erc20/test_erc20.cairo

```diff
@@ -162,12 +163,13 @@ fn test_transfer() {
     let mut spy = spy_events();
 
     start_cheat_caller_address(contract_address, OWNER());
-    assert!(state.transfer(RECIPIENT(), VALUE));
+
+    assert_state_before_transfer(OWNER(), RECIPIENT());
+    let success = state.transfer(RECIPIENT(), VALUE);
+    assert!(success, "Transfer failed");
```

Comment:
I don't think this is error message is adding real value, you guys think it does?

Reply:
Some testing practices recommend to follow the rule "one test case — one assertion". If it was the case for our tests, then adding an error message would make no difference. But in most cases we have 3+ assertions and an error occurs in a test, then it may be difficult to find out which assertion has failed